cfgov/unprocessed/css/pages/ask.less

```diff
@@ -65,4 +62,58 @@
             }
         }
     }
-}
\ No newline at end of file
+
+    
+}
+
+/* Results pages */
+
+.ask-category-page {
+    h1 .cf-icon {
+        color: @green;
+    }
+
+    .lead-paragraph {
+        max-width: 600px;
```

Comment:
I'm not sure why this is necessary, but if we are using it repeatedly then it make make sense to make it a variable.